src/isomorphic/classic/element/__tests__/ReactElement-test.js

```diff
@@ -24,6 +24,10 @@ describe('ReactElement', function() {
   beforeEach(function() {
     require('mock-modules').dumpCache();
 
+    // Delete the native Symbol if we have one to ensure we test the
+    // unpolyfilled environment.
+    delete global.Symbol;
```

Comment:
Well Symbol isn't from jsdom, right? Could add an `expect(typeof Symbol).toBe('undefined');` right after to be safe.

Also: we should probably restore the old Symbol global after this test.